src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import duke.Deadline;
+import duke.Event;
+import duke.Task;
+import duke.Todo;
+import exception.DukeException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    static List<Task> tasks = new ArrayList<>();
+    static Boolean notBye = true;
```

Comment:
As good coding practice, you could indicate the access modifiers (`public`, `private`, `protected` etc..) for these variables.